lib/locale/pl/strings.php

```diff
@@ -24,6 +24,9 @@
  *  $Id$
  */
 
+$_LANG['Verified'] = 'Weryfikowany';
+$_LANG['Ticket has been transferred to verifier.'] = 'Zgłoszenie zostało wysłane do weryfikatora.';
```

Comment:
czy raczej ... zostało przekazane do ... ?

Reply:
To teraz lepiej
"Ticket has been transferred to verifier."
czy 
"Ticket has been set for verification"
?

Reply:
Ticket has been transferred to verifier.